config/nginx.conf.erb

```diff
@@ -18,7 +18,20 @@ http {
     server_tokens off;
 
     log_format l2met 'measure#nginx.service=$request_time request_id=$http_x_request_id';
-    access_log logs/nginx/access.log l2met;
+    log_format apm '"$time_local" client=$remote_addr '
+                   'method=$request_method request="$request" '
+                   'request_length=$request_length '
+                   'status=$status bytes_sent=$bytes_sent '
+                   'body_bytes_sent=$body_bytes_sent '
+                   'referer=$http_referer '
+                   'user_agent="$http_user_agent" '
+                   'upstream_addr=$upstream_addr '
+                   'upstream_status=$upstream_status '
+                   'request_time=$request_time '
+                   'upstream_response_time=$upstream_response_time '
+                   'upstream_connect_time=$upstream_connect_time '
+                   'upstream_header_time=$upstream_header_time';
+    access_log logs/nginx/access.log apm;
```

Comment:
It's not required, but we can if it would be valuable for developers to have that extra info. Most of that detail is already captured in the default Nginx logs for that container.